vdr/types/interface.go

```diff
@@ -35,6 +35,9 @@ type DocResolver interface {
 	// If metadata is provided then the result is filtered or scoped on that metadata.
 	// It returns ErrNotFound if there are no corresponding DID documents or when the DID Documents are disjoint with the provided ResolveMetadata
 	Resolve(id did.DID, metadata *ResolveMetadata) (*did.Document, *DocumentMetadata, error)
+	// ResolveControllers accepts a list of documents and finds their controllers
+	// The resulting list are documents who control themselves
```

Comment:
The addition of "The resulting list are documents who control themselves" can now be removed I think.